Index: virtuoso-opensource/binsrc/VirtuosoClient.Net/Makefile.mono
===================================================================
--- virtuoso-opensource.orig/binsrc/VirtuosoClient.Net/Makefile.mono	2012-05-02 16:10:10.835193476 +0200
+++ virtuoso-opensource/binsrc/VirtuosoClient.Net/Makefile.mono	2012-05-02 16:11:10.503195047 +0200
@@ -86,7 +86,7 @@
 
 ifeq  (ADONET2, $(ADONETVERSION))
 
-CSC		= gmcs
+CSC		= mono-csc
 
 #CSFLAGS	= -debug+ /d:MONO /d:DEBUG /d:DEBUG_IO /d:ADONET2
 CSFLAGS		= /optimize+ /d:MONO /d:ADONET2
@@ -101,7 +101,7 @@
 
 else
 
-CSC		= mcs
+CSC		= mono-csc
 
 #CSFLAGS	= -debug+ /d:MONO /d:DEBUG /d:DEBUG_IO
 CSFLAGS		= /optimize+ /d:MONO
